contrib/dev/Vagrantfile

```diff
@@ -107,19 +128,47 @@ def populate_os_config(os, ip)
       address: '192.168.10.10',
     },
   ]
+  os.security_groups = [ 'skydive' ]
+end
+
+def fedora_box(override, type)
+  major = "28" 
+  minor = "1.1"
+  override.vm.box = "fedora/#{major}-cloud-base"
+  override.vm.box_url = "http://mirror.isoc.org.il/pub/fedora/releases/#{major}/Cloud/x86_64/images/Fedora-Cloud-Base-Vagrant-#{major}-#{minor}.x86_64.vagrant-#{type}.box"
+end
+
+def synced_folder(override, type, host, guest)
+  if type == "nft" then
+    override.vm.synced_folder host, guest, type: "nfs", nfs_udp: false
+  end
+  if type == "virtualbox" then
+    override.vm.synced_folder host, guest, type: "virtualbox", user: "vagrant", group: "vagrant"
+  end
+end
+
+def dev_synced_folders(override, type)
+    if SKYDIVE_SYNC_FOLDER then
+      synced_folder(override, type, SKYDIVE_HOST, SKYDIVE_GUEST)
+    end
```

Comment:
nit: elsif type == "virtualbox" then

Reply:
the ansible folder is independently needed - so I am not sure an `elsif` if correct here